test/test_testing.py

```diff
@@ -413,6 +413,15 @@ def test_get_supported_dtypes(self, device):
             self.assertTrue(set(dtypes) == set(dynamic_dtypes))
             self.assertTrue(set(dtypes) == set(dynamic_dispatch.dispatch_fn()))
 
+    @onlyCPU
+    def test_supported_dtypes(self, device):
+        for op in op_db:
+          if len(op.supported_dtypes('cpu').symmetric_difference(op.supported_dtypes('cuda'))) > 0:
+            break
```

Comment:
Nit:

```suggestion
    @ops(
        [op for op in op_db if len(op.supported_dtypes("cpu").symmetric_difference(op.supported_dtypes("cuda"))) == 0],
        dtypes=OpDTypes.none,
    )
    def test_supported_dtypes(self, device, op):
        pass
```

If you go for this, you also need to unindent the checks below one level, since we removed the loop.